schema/vrs/vrs-source.yaml

```diff
@@ -343,6 +343,17 @@ $defs:
         enum:
           - aa
           - na
+      sequence:
+        type:
+          $ref: "#/$defs/SequenceString"
+        description: >-
+          A :ref:`SequenceString` that is a literal representation of the referenced sequence.
+      moleculeType:
```

Comment:
@cmprocknow concern about "molecule type"; perhaps "molecularContext" would be cleaner.

Reply:
I think we can sort this out during public review.

Reply:
RefSeq calls this Molecule Type, so seems appropriate to keep.